aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -599,6 +599,109 @@ Tensor& diff_out(const Tensor& self, int64_t n, int64_t dim, const c10::optional
   }
 }
 
+void pre_check_gradient(const Tensor& self, IntArrayRef axis, int64_t spacing_size,  int64_t edge_order) {
+  TORCH_CHECK(self.scalar_type() != ScalarType::Byte, "torch.gradient does not support uint8 as input.");
+  TORCH_CHECK( spacing_size == axis.size(), "torch.gradient should receive axis and spacing size to be equal.");
+  TORCH_CHECK( edge_order == 1, "torch.gradient supports edge_order value only to be equal to 1.");
```

Comment:
"torch.gradient supports edge_order value only to be equal to 1." -> "torch.gradient only supports edge_order=1 currently. Please file an issue at https://github.com/pytorch/pytorch/issues/new?assignees=&labels=&template=feature-request.md to request support for other edge_order values."